components/org.wso2.carbon.identity.oauth.par/src/main/java/org/wso2/carbon/identity/oauth/par/exceptions/ParClientException.java

```diff
@@ -18,17 +18,18 @@
 
 package org.wso2.carbon.identity.oauth.par.exceptions;
 
+import org.wso2.carbon.identity.base.IdentityException;
+
 /**
  * Client exception from par core component.
  */
-
-public class ParClientException extends ParCoreException {
+public class ParClientException extends IdentityException {
```

Comment:
Extend the client exception from the server exception(ParCoreException). This will be easy for error handling.